src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -136,6 +137,12 @@ public String key() {
   }
 
   static class PlaybackDelegate implements MediaHeartbeatDelegate {
+    long initialTime;
```

Comment:
the initialTime should only be set once - during construction. I'd move it to the constructor and make it final - you don't need to check/set it in getCurrentPlaybackTime